### PR TITLE
migrate to xcode 10.1

### PR DESCRIPTION
**Pull request by niil-ohlin**

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -71,7 +69,6 @@ case "$COMMAND" in
     -scheme "${SCHEME}" \
     -sdk "${IOS_SDK}" \
     -destination "${IOS_DESTINATION_PHONE}" \
-    -destination "${IOS_DESTINATION_PAD}" \
```
Comment (fennecos) [inline build.sh]: Is this change intentional?

Comment (niil-ohlin) [reply build.sh]: Yes for some reason xcodebuild10.1 could not run with multiple destinations. Is it important to keep?

Comment (fennecos) [reply build.sh]: It's fine I guess
